src/ntttcp.c

```diff
@@ -2,7 +2,10 @@
 Copyright (c) Microsoft Corporation.
 */
 
-#define NTTTCP_VERSION "5.39"
+#define NTTTCP_VERSION "5.40" // TODO: Replace with VER_MAJOR "." VER_MINOR

```

Comment:
can we go ahead and do this? As is, now there are two points of maintenance- here and the new json file

Reply:
This requires changes to the undocked repo. I will do it soon, but don't want to block here.